style.css

```diff
@@ -1,9 +1,331 @@
+}
+
+.photo {
+    padding-bottom: 65px;
+
+}
+
+.btn {
+    border-radius: 32px;
+    padding: 9px 30px;
+    font-size: 16px;
+    background-color: #D9DCD7;
+    
+}
+
+/* TODO: order styles by sections/content */
+
+.section_title{
+  /* now I set margins on this element to match figma template spacing */
+  margin: 88px 0;
+  font-size: 36px;
+  font-weight: 400;
+  font-size: 36px;
+  text-align: center;
+  line-height: 119.5%;
+}
+
+.all_pros_container{
+    display: flex;
+    justify-content: center;    
+}
+
+.high_way{
+    margin: 0px 8px;
+}
+
+
+.paragraph_header{
+    font-weight: bold;
+    
+    
+}
+.pros_photo{
+    width: 400px;
+    height: 324px;
+}
+
+.paragraph_description{
+    width: 400px;
+    display:flex;
+    justify-content:center;
+    
+}
+.header_container{
+    display:flex;
+    justify-content: center;
+    width: 400px;
+}
+
+.container_for_paragraph{
+    height: 200px;
+    display: flex;
+    justify-content: center;
+}
+
+.text{
+    opacity: 0.8;
+    text-align: center;
+}
+
+
+
+
+.third_page_container{
+    display: flex;
+}
+
+.left_side{
+    display: flex;
+    height: 811px;
+    width: 712px;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    background-color: #ECE0E0;
+}
+
+.right_side{
+    height: 811px;
+    width: 712px;
+}
+.mountains_pic{
+    height: 811px;
+    width: 712px;
+}
+
+.some_text{
+    width: 400px;
+    height: 220px;
+    text-align: left;
+    font-family: 'Roboto', sans-serif;
+}
+.longer_paragraph{
+    opacity: 0.7;
+    font-size: 17px;
+}
+.third_page_header{
+    padding-left: 8px;
+    font-size: 25.5px;
+    font-weight: normal;
+}
+
+.btn_third_sector{
+    width: 400px;
+}
+
+.book_now{
+    border-radius: 32px;
+    width: 130px;
+    height: 35px;
+    background-color: #ECE0E0;
+    
+}
+
+.book{
+font-size: 16px;
+}
+
+.fourth_page_container{
+    display: flex;
+}
+
+.girl_pic{
+    width: 504px;
+    height: 604px;
+}
+.left_side_fourth_page{
+    display:flex;
+    height: 704px;
+    align-items: flex-end;
+}
+.right_side_fourth_page{
+    width: 936px;
+    height: 704px;
+    display: flex;
+    flex-direction: column;
+    
+    
+}
+.fan_number{
+    width: 936px;
+    height: 302px;
+    display: flex;
+    align-items:center;
+   
+    
+}
+
+.cabin_lovers{
+    font-size: 48px;
+    font-weight: normal;
+    font-family: 'Roboto', sans-serif;
+    margin-block-start: 0em;
+    margin-block-end: 0em;
+    margin-inline-start: 100px;
+    margin-inline-end: 0px;
+    
+}
+.autor{
+    width:936px;
+    height:402px;
     display:flex;
+    align-items: flex-end;
+    margin-left: 100px;
+}
+.fifth_page_container{
+    display:flex;
+    height: 300px;
+    width:936px;
+}
+.newsletter{
+  display:flex;
+  width:468px;
+  align-items: center;
+  padding-left: 32px;
+}
+.input_newsletter{
+width:468px;
+}
+
+.input_header{
+    padding-bottom: 56px;
+}
+
+.newsletter_info{
+    opacity: 0.7;
+}
```

Comment:
the opacity should be 1, fix the colors ;)

Reply:
still not done ⛔

Reply:
fixed